example/src/util.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { ReactCornerstoneViewportHooksHelpers } from "react-cornerstone-viewport-hooks";
+
+export const useDebugToolState = () => useEffect(() => {
```

Comment:
print toolstate to console quite important for our backend,
added beecause sometimes was not stored correctly